cms/search/__init__.py

```diff
@@ -0,0 +1,13 @@
+from typing import Union
+
+from django.conf import settings
+
+from .publishers import KafkaPublisher, LogPublisher
+
+
+def get_publisher() -> Union[KafkaPublisher, LogPublisher]:
```

Comment:
also, it may be a good idea to cache the publisher. [reference](https://adamj.eu/tech/2023/03/02/django-profile-and-improve-import-time/#do-other-kinds-of-work-lazily)

Reply:
Done!